src/renderers/dom/client/validateDOMNesting.js

```diff
@@ -327,8 +327,8 @@ if (__DEV__) {
     var parentInfo = ancestorInfo.current;
     var parentTag = parentInfo && parentInfo.tag;
 
-    var invalidParent =
-      isTagValidWithParent(childTag, parentTag) ? null : parentInfo;
+    var invalidParent = childInstance._isWhitespace ?
```

Comment:
Will skip valid parent check for all children deemed whitespace. Not just elements in `<table>`